firstname.lastname.txt

```diff
@@ -0,0 +1,5 @@
+Jonathan Krilosky
+I am okay with being addressed as Jon
+I am in section 294
+I prefer Java, but I am still learning Java as it is. I am open to learning new tech stack as well
+Something to know is I am still somewhat of a newbie
```

Comment:
Thanks Jon -
Please make sure to clean up your own repository to avoid exposing any private information to the public